docs/Using Randomness/get-a-random-number.md

```diff
@@ -160,3 +168,8 @@ After you are done with this contract and the subscription, you can retrieve the
 1. Under your subscription details, click **Cancel subscription**. A field opens asking which wallet address you want to send the remaining funds to.
 
 1. Enter your wallet address and click **Cancel subscription**. MetaMask opens and asks you to confirm the transaction. After you approve the transaction, Chainlink VRF closes your subscription account and sends the remaining LINK to your wallet.
+
+## Vyper Example
+
+You'd need to import the `VRFCoordinatorV2` vyper interface, you can find it  [here](https://github.com/smartcontractkit/apeworx-starter-kit/blob/main/contracts/interfaces/VRFCoordinatorV2.vy).
```

Comment:
```suggestion
You must import the `VRFCoordinatorV2` vyper interface. You can find it  [here](https://github.com/smartcontractkit/apeworx-starter-kit/blob/main/contracts/interfaces/VRFCoordinatorV2.vy).
```

Reply:
Also, is Vyper captialized? I wasn't too sure

Reply:
@ritika-b  please approve :)